platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -494,6 +494,14 @@ namespace platoon_strategic_ihp
                 status_msg.host_platoon_position = pm_.getNumberOfVehicleInFront();
                 ROS_DEBUG_STREAM("pm platoonsize: " << pm_.getTotalPlatooningSize() << ", platoon_leader " << platoon_leader.staticId);
 
+                int numOfVehiclesGaps = pm_.getNumberOfVehicleInFront() - pm_.dynamic_leader_index_;
```

Comment:
The naming seems pretty clear, but I'm not sure if this is doing what it appears to be.  If host is 3rd car, then getNumberOfVehiclesInFront() should return 2, yes?  And dynamic_leader_index_ would normally be 1 (the immediately preceding vehicle), yes?
Maybe worth a discussion.

Reply:
The hostPosInPlatoon_ is a private member. Aslo, what you are suggesting is resulting in negative value?